template/koa-rest-api/Dockerfile

```diff
@@ -1,15 +1,14 @@
-ARG BASE_IMAGE
-ARG BASE_TAG
+FROM node:16-alpine AS deps
 
-###
+WORKDIR /workdir
 
-FROM ${BASE_IMAGE}:${BASE_TAG} AS deps
+COPY package.json yarn.lock ./
 
-RUN yarn install --ignore-optional --ignore-scripts --non-interactive --offline --production
+RUN yarn install --ignore-optional --ignore-scripts --non-interactive --production
 
 ###
 
-FROM ${BASE_IMAGE}:${BASE_TAG} AS build
+FROM deps AS build
```

Comment:
This will need to be `FROM dev-deps as build` as we need `dev-deps` like skuba itself to be able to run the build (while prod `deps` only will be copied to the final runtime stage)